sklearn/svm/classes.py

```diff
@@ -704,6 +714,14 @@ class NuSVC(BaseSVC):
         .. versionchanged:: 0.17
            Deprecated *decision_function_shape='ovo' and None*.
 
+    break_ties : bool, optional (default=False)
+        If true, `decision_function_shape`='ovr', and number of classes > 2,
```

Comment:
the practical utility from this explanation may not be clear to everybody

Reply:
any suggestions?

Reply:
We can put in a very common use case that a wide spectrum of data analysts are likely to have encountered in some shape or form.

Reply:
those details are suitable for .rst documentations, not the docstrings.

Reply:
Actually, I must confess that I am unsure what exactly the benefits of turning this on is. Let me explain what I think I have understood that the benefit is, and maybe it can help the docstring:
"Using confidence to break ties can lead to a better prediction for this specific instances".

Is that correct? If so, I would suggest that this sentence is added before "Please note that..."